testing.rst

```diff
@@ -535,6 +535,15 @@ This allows you to create all types of requests you can think of:
     :ref:`framework.test <reference-framework-test>` option is enabled).
     This means you can override the service entirely if you need to.
 
+.. caution::
+
+    Before each request, the client reboots the kernel, recreating the container from scratch.
```

Comment:
Can you please be careful to break lines at 80 characters.

Reply:
Done